### PR TITLE
Champs Code Review

### PR DESCRIPTION
**Pull request by andrewfhou**

### CODE REVIEW

**Review by eku952 — APPROVED** [review]
send it

**Review by Aryan9101 — APPROVED** [review]
Looks good to me

File: src/main/java/frc/team1816/robot/Robot.java [hunk 1]
```diff
@@ -60,30 +63,41 @@ public void robotInit() {
...
+        modScaleAuto = new ModdedScaleAutoCommand();
 
         startPosition = new SendableChooser<>();
         startPosition.addObject("Left Start", "Left Start");
         startPosition.addObject("Right Start", "Right Start");
         SmartDashboard.putData("Start Position", startPosition);
 
         autoChooser = new SendableChooser<>();
+
+        autoChooser.addObject("Qual 26 Auto (modded scale)", modScaleAuto);
```
Comment (MGuron) [inline src/main/java/frc/team1816/robot/Robot.java]: Should probably remove all instances of this or repurpose

**Review by MGuron — APPROVED** [review]
Looks great, Can't wait to get back to tuning autos for the last piece of accuracy.

**Review by andrewfhou — COMMENTED** [review]
Auto's working as of Detroit (DO NOT CHANGE): Far scale regular (in qual 26 auto)

Near avoidance (in qual 26 auto) mostly functional, minimal tweaking.

TO WORK ON: Center Switch 2-cube, far avoidance, tweak near-avoidance

**Pull request merged**